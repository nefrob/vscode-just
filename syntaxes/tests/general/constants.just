# Constants

true
false

1234
1234.
12.34
0.1234
.1234

HEX
HEXLOWER
HEXUPPER
