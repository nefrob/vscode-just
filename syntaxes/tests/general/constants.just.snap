># Constants
#^^^^^^^^^^^ source.just comment.line.number-sign.just
>
>true
#^^^^ source.just constant.language.boolean.just
>false
#^^^^^ source.just constant.language.boolean.just
>
>1234
#^^^^ source.just constant.numeric.just
>1234.
#^^^^^ source.just constant.numeric.just
>12.34
#^^^^^ source.just constant.numeric.just
>0.1234
#^^^^^^ source.just constant.numeric.just
>.1234
#^^^^^ source.just constant.numeric.just
>
>HEX
#^^^ source.just constant.language.hex.just
>HEXLOWER
#^^^^^^^^ source.just constant.language.hex.just
>HEXUPPER
#^^^^^^^^ source.just constant.language.hex.just
>